experimental/Schemes/SpecOpen.jl

```diff
@@ -204,57 +204,50 @@ function intersect(
     check::Bool=true
   )
   X = ambient(U)
-  ambient_ring(X) === ambient_ring(Y) || error("Schemes can not be compared")
+  ambient_ring(U) === ambient_ring(Y) || error("schemes can not be compared")
```

Comment:
```suggestion
  ambient_ring(X) === ambient_ring(Y) || error("schemes can not be compared")
```
Do we even have `ambient_ring` for `SpecOpen`s ?

Reply:
yes